ci/publish-packages.sh

```diff
@@ -35,6 +35,22 @@ function getSecrets() {
         # leave it in the format that cloudsmith expects. See https://github.com/cloudsmith-io/cloudsmith-cli#configuration
         sed -i '1s/^/[default]\napi_key=/' "${CLOUDSMITH_CONFIG_FILE}"
 
+        export NPMRC_DIR=$(mktemp -d "${TMPDIR:-/tmp}/XXXXXXXXX")
+        local NPMRC_FILE="${NPMRC_DIR}/npmrc"
+
+        # TODO: Read secret and setup .npmrc file to mount into the docker container
+        imp-ci secrets read \
+            --environment="production" \
+            --secret-type="generic-token" \
+            --buildkite-org="improbable" \
+            --secret-name="unity/npm-spatialos-china-dogbot" \
+            --field="token" \
+            --write-to="${NPMRC_FILE}"
+
+        # We don't want to write the API key to the console, so ewe use sed to append the contents to the front of the file to
```

Comment:
```suggestion
        # We don't want to write the API key to the console, so we use sed to append the contents to the front of the file to
```

Reply:
<details>
<summary>But ewe is using sed!</summary>

![image](https://user-images.githubusercontent.com/13353733/85707036-b5f3c900-b6da-11ea-893e-4c5e8535476b.png)

</details>